src/parser/metrics/RealLinesOfCode.ts

```diff
@@ -40,25 +40,33 @@ export class RealLinesOfCode implements Metric {
         isComment: (node: Parser.SyntaxNode) => boolean,
         realLinesOfCode = new Set<number>()
     ) {
+        const currentNode = cursor.currentNode;
         // This is not a comment syntax node, so assume it includes "real code".
```

Comment:
we can delete this comment i think. It's clear enough what happens here.

Reply:
I have removed most comments here now